AutoRest/Generators/Ruby/Azure.Ruby/Templates/AzureMethodTemplate.cshtml

```diff
@@ -27,29 +26,19 @@ else if (Model.HttpMethod == HttpMethod.Post || Model.HttpMethod == HttpMethod.D
 {
 @:@WrapComment("# ", string.Format("@param {0} {1}{2}", parameter.Name, parameter.Type.GetYardDocumentation(), parameter.Documentation))
 }
-@WrapComment("# ", "@return [Concurrent::Promise] promise which provides async access to http response.")
+@WrapComment("# ", string.Format("@param custom_headers [{0}] A hash of custom headers that will be added to the HTTP request.", "Hash{String => String}"))
 #

```

Comment:
why do we need to manually specify 'custom_headers' parameter? Isn't it already in LocalParameters?